assin.php

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title></title>
+</head>
+<body>
+
+
+<!-- 1st assignment -->
+<?php
+$f_name =  $_POST["fname"];       //store first name via post method
+$l_name =  $_POST["lname"];       //store last name via post method
+$full_name = $f_name." ".$l_name; //concatenate the fname and lname
+?>
+
+Hello <?php echo $full_name;?><br>   <!-- print the full name in web page -->
+
+
+<!-- 2nd assignment -->
+<?php
+
+    // file name with extention
+    $file = $_FILES['file']['name'];
+
+    // file's original path
+    $temp_name = $_FILES['file']['tmp_name'];
+
+    // path to move the uploaded file
+    $path = "uploads/".$file;
+
+    // move the uploaded file to the upload folder
+    move_uploaded_file($temp_name,$path);
+
+?>
+image : <br>
+<img src="uploads/<?php echo $file?>" height="300px" width="200px" >
```

Comment:
Suppose if the user does not upload the image, then should we add this image tag in our DOM? Also try to add alt and title attributes for image tags, these are important for SEO purpose.